### ISSUE TITLE
Conditional reveal aria-controls attribute added with JS

### ISSUE THREAD

**Issue by adamsilver** [alphagov/govuk-frontend#685]
Currently, the aria-controls attribute is added during JS initialisation.

My thinking is that the attribute should be in the HTML regardless. 

This way when a screen reader user focuses the radio button (or checkbox) they will be able to move to the controlled panel using additional shortcuts.

**Comment by joshueoconnor** [comment]
+1 to @adamsilver 

Actually AFAIK, the use of these kinds of ARIA attributes/properties is not JavaScript dependent at all but on support within the user agent (whether browser or AT). The points that Leonie makes on the blog post above are where JS is used as the delivery mechanism for 'some useful ARIA thing that makes something more accessible' - which is of course, a valid point.

**Comment by nickcolley** [comment]
https://www.w3.org/WAI/PF/aria-1.1/states_and_properties#aria-controls

> Identifies the element (or elements) whose contents or presence are controlled by the current element.

We use aria-controls on these elements to enhance aria-expanded, which is [not an allowed attribute for checkboxes and radio buttons](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties#aria-expanded).

So I think really it's perhaps a bit of a stretch to use aria-controls in a scenario where the content is static. For pragmatic reasons adding this the to markup when we're considering removing aria-controls and aria-expanded would mean we'd have to make a breaking change in the future.
Finally, assistive technologies have limited support for this attribute when used on these elements.

So with all this in mind I'll close this out for now but if something changes and [we do not decide to remove these attributes then we can reconsider this](https://github.com/alphagov/govuk-frontend/issues/979).
